oracle/price-feeder/oracle/oracle.go

```diff
@@ -521,6 +528,10 @@ func (o *Oracle) getOrSetProvider(ctx context.Context, providerName string) (pro
 		ok            bool
 	)
 
+	if err, ok := o.failedProviders[providerName]; ok {
```

Comment:
can we add a TODO here to replace this behavior with a retry with backoff in the future?